### PR TITLE
Add bedrock data to pages

### PR DESCRIPTION
**Pull request by extremeheat**
The bedrock protocol data is generated as static HTML, so it should load instantly. Right now the bundle.js is 60MB so this should probably be addressed.

ported from https://github.com/PrismarineJS/bedrock-protocol/pull/98

### CODE REVIEW

File: loadProtocol.js [hunk 1]
```diff
@@ -293,9 +293,14 @@ function eqs (compareTo, k) {
 }
 
 function loadProtocol (version) {
-  const data = require('minecraft-data')(version).protocol
-  const comments = require('minecraft-data')(version).protocolComments
-  $j('#protocolTable').html(protocolToString(data, comments))
+  if (version.startsWith('bedrock')) {
+    const [, v] = version.split('_')
+    $j('#protocolTable').html(`<iframe src="protocol/bedrock/${v}" frameborder="0" style='width: 100%; height: 80vh;'><a href="protocol/bedrock/${v}">Click here</a></iframe>`)
```
Comment (rom1504) [inline loadProtocol.js:298]: Do we really need an iframe ?

Comment (extremeheat) [reply loadProtocol.js:298]: The html could be loaded into the page with jquery, however the styles used in the html file which is self contained could collide with the page styles so I used an iframe here.

Comment (rom1504) [reply loadProtocol.js:298]: why can't we use browserify like the rest of this package ?

Comment (rom1504) [reply loadProtocol.js:298]: ah it's this compiler thing from protodef yaml hm

Comment (rom1504) [reply loadProtocol.js:298]: yeah well still... there has to be a better way than an iframe...

**Comment by rom1504** [comment]
so actually about the iframe I don't care that much as long as it works
I tried it `npm run prod-start` and I don't see any bedrock link anywhere
should it work like this ?

File: loadProtocol.js (hunk 1 above)
Comment (extremeheat) [reply loadProtocol.js]: I took a look at removing the header and putting in the css here, but the issue is that it uses bootstrap and without it it looks a lot uglier IMO.

**Comment by extremeheat** [comment]
Fixed

**Comment by rom1504** [comment]
ah I just realized that this PR is targeting the wrong branch
this should be targeting the *web* branch, not gh-pages

**Comment by extremeheat** [comment]
Updated

**Comment by rom1504** [comment]
so I tested it and it looks really good
one issue is there is no scroll bar in the protocol page

there are probably several ways to fix it

one could be to use a web component (https://lit-element.polymer-project.org/guide) to keep the css encapsulated while not using an iframe

**Comment by extremeheat** [comment]
Ah, probably an issue with the styling. Wouldn't adding lit element add extra tooling?

I'm not really familiar with web components but it [seems like it could work yeah](https://sebastiandedeyne.com/embed-a-web-page-with-a-web-component-and-the-shadow-dom/) with a shadow DOM, I'll take a look.

**Comment by rom1504** [comment]
no lit element is just about adding a dep

**Comment by extremeheat** [comment]
OK, I've switched it to use a web component. Since the shadow DOM doesn't get the same events as the parent, clicking on the anchor links didn't do anything (no scrolling) so I wrote a simple handler to proxy over the events.

**Comment by rom1504** [comment]
there's some lint issue

**Comment by extremeheat** [comment]
fixed

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
thanks

I noticed the anchors do not work when using a link with anchors
not sure why

**Comment by extremeheat** [comment]
Ah yeah, I think that can be fixed by calling `scrollCallback` in the connectedCallback

**Pull request merged**